ui/src/admin/components/AdminTabs.js

```diff
@@ -1,11 +1,5 @@
 import React, {PropTypes} from 'react'
-import {
-  Tab,
-  Tabs,
-  TabPanel,
-  TabPanels,
-  TabList,
-} from 'shared/components/Tabs'
+import {Tab, Tabs, TabPanel, TabPanels, TabList} from 'shared/components/Tabs'
```

Comment:
No, I should have linked to the 1.4 changelog. These changes make sense once you review it. One moment, I'll update the description of the PR with my settings screenshot and the 1.4 changelog.